### PR TITLE
[hotfix] Download as zip button does not appear for closed folders

### PR DESCRIPTION
**Pull request by reinaH**
# Purpose: 
- When the user clicks on (but does not open) a closed folder that contains files it is expected that a "Download as zip" button should appear. 

# Changes: 
-Change in fanghorn.js

# Side Effects:
Users will be able to download empty folders. This side effect was discussed with @erinmayhood and the resulting issue is addressed in a new JIRA ticket: https://openscience.atlassian.net/browse/OSF-4549

# Notes: 
Closes: 
- https://github.com/CenterForOpenScience/osf.io/issues/3388
- JIRA ticket: https://openscience.atlassian.net/browse/OSF-3429

### CODE REVIEW

File: website/static/js/fangorn.js [hunk 1]
```diff
@@ -1527,7 +1527,7 @@ var FGItemButtons = {
                         ])
                     );
                 }
-            } else if (item.data.provider && item.children.length !== 0) {
+            } else if (item.data.provider && item.open.length !== 0) {
```
Comment (sloria) [inline website/static/js/fangorn.js]: Ping @caneruguz for review; is `item.open` an Array or Boolean?

Comment (reinaH) [inline website/static/js/fangorn.js]: @sloria boolean. ill fix it.

**Comment by HarryRybacki** [comment]
Looks good to me! CR 1 :+1:

**Comment by samanehsan** [comment]
Looks good to me @reinaH! Done with CR2 :octocat:

**Pull request merged**